app/processing/statement/get-statement.js

```diff
@@ -16,7 +17,9 @@ const getStatement = async (settlementId) => {
     const address = await getAddress(sbi, transaction)
     const detailedFunding = await getDetailedFunding(calculation.calculationId, paymentRequestId, transaction)
     const scheme = await getScheme(paymentRequest)
-    const payments = await getDetailedPayments(calculation, paymentRequest, settlement)
+    const lastSettlement = await getLastSettlement(settlement, transaction)
+    const latestPayment = getLatestPayment(paymentRequest, settlement, lastSettlement)
+    const payments = await getDetailedPayments(calculation, latestPayment, settlement, lastSettlement)

```

Comment:
`getDetailedPayments` here takes 4 params but its definition it only takes 3

Reply:
Should be a test case covering what  `getDetailedPayments` is called with which should have failed too